lib/ActivityStreams.js

```diff
@@ -113,6 +114,12 @@ function ActivityStreams(metadataStore, options = {}) {
   this._setupButton();
   NewTabURL.override(this._newTabURL);
   this._setHomePage();
+  this.prefsProvider = new PrefsProvider({
```

Comment:
you're right, it should be `_` (we use that for "private" properties, albeit somewhat inconsistently)